### DISCUSSION TITLE
【SEO-Google图片搜索】网站的Google图片搜索结果很糟糕，没有匹配为封面

### DISCUSSION THREAD

**Discussion by DoiiarX** [tangly1024/NotionNext#1998] [Q&A]
**描述bug**
网站的Google图片搜索结果很糟糕，没有匹配为封面
Google收录之后封面是无关的图片
![image](https://github.com/tangly1024/NotionNext/assets/25550075/691111cc-eb39-4926-b4fe-905a20ce5164)
但是我实际上已经设置了封面
![image](https://github.com/tangly1024/NotionNext/assets/25550075/39e1e66a-665d-454e-ab67-90b86cea8925)

**复现步骤**
发布网页、甚至封面、然后被Google收录

**期望的正常结果**
![image](https://github.com/tangly1024/NotionNext/assets/25550075/03d2a3f9-e353-4be0-836a-69971675ed65)
以正确的封面被Google收录
**截图**
【可选】相关的页面，应该的结果

**环境**

- 【必填】NotionNext版本 4.1.5
- 【必填】主题 hexo
- 【可选】操作系统: windows
- 【可选】浏览器 edge

**补充说明**
【可选】与问题相关的其它说明
无

**Comment by tangly1024** [comment]
刚刚搜索了一下确实出现此问题，但搜索引擎通常只抓取页面内的图片，至于这些图片是怎么出现的，我需要再观察一下

![image](https://github.com/tangly1024/NotionNext/assets/15920488/64e5fd41-f21e-453f-a0ac-f39a0a3ddad6)
